### DISCUSSION TITLE
I am new and i need help with getting Kelper and TESS files

### DISCUSSION THREAD

**Discussion by Ribel1009** [lightkurve/lightkurve#1242] [Q&A]
i am learner about lightkurve and i didn t understand something that s why i came to ask you to help me please

so what i didn t understand is that when i want to work with a new star i will need the quarter if it s from kepler  catalog or a sector if it s from the TESS but i don t know how to get this information and how to know if my objects isa TESS or a Kepler

**Comment by orionlee** [comment]
See the following lightkurve tutorial on how to do so:
https://docs.lightkurve.org/tutorials/1-getting-started/searching-for-data-products.html#2.-Searching-for-Light-Curves

Basically, given a star, `search_lightcurve()` can return a list of all available lightcurves, across all quarters/sectors.

**Comment by Ribel1009** [comment]
well thank you very much that solved my problem with the kepler objects but
what about the TESS ones how can i search for them on the MAST and get
their TESS name like "Pi Men " to use it later in the code
*B.Ribel Boujlida*

**Comment by orionlee** [comment]
The principal of the tutorial applied to TESS data  as well. If you want to search for pi Men, 

```python
search_result = lk.search_lightcurve("pi Men", mission="TESS")
```

If you don't care whether it's from TESS or Kepler (some targets do appear on both), simply omit the optional mission parameter:

```python
search_result = lk.search_lightcurve("pi Men")
```

**Comment by Ribel1009** [comment]
oh.. that s great but how can i get the pi Men from i mean from where can i
get the name like that if i want to use a new star data ?

**Reply by orionlee** [reply]
I'm unsure what you're looking for... Usually one has some target(s) in mind (or some other criteria) and download the relevant data. It sounds like you want to browse for some exoplanets. If so, here are some resources:

- NASA Exoplanet exploration https://exoplanets.nasa.gov/discovery/exoplanet-catalog/ . Note not all of them observed by TESS.
- ExoFOP maintains a list of TOIs (TESS Object of Interests), basically a list of planet candidates:  https://exofop.ipac.caltech.edu/tess/view_toi.php
